sdk/clientcore/platform-matrix.json

```diff
@@ -15,8 +15,8 @@
       "NotFromSource_TestsOnly": {
```

Comment:
Do we need to update the `"TEST_HTTP_CLIENT_IMPLEMENTATION": [ "io.clientcore.core.http.client.DefaultHttpClientProvider"` to Java12 DefaultHttpClientProvider?

Reply:
Nope, they use the same name. Just the non-working code in Java 8 and 11 gets overridden by the Java 12 implementation when running with Java 17 and 21.